src/api/Jalasoft.TeamUp.Projects.API/Controllers/GetProjects.cs

```diff
@@ -0,0 +1,32 @@
+namespace Jalasoft.TeamUp.Projects.API.Controllers
+{
+    using System.Net;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.OpenApi.Models;
+
+    public class GetProjects
+    {
+        private readonly IProjectsService projectService;
+
+        public GetProjects(IProjectsService projectService)
+        {
+            this.projectService = projectService;
+        }
+
+        [FunctionName("projects")]
+        [OpenApiOperation(operationId: "Run", tags: new[] { "Projects" })]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Project), Description = "Successful response")]
+        public IActionResult Run(
```

Comment:
I'm curios, why not static method/classes for the API layer? @cristian-chavez-jalafund @Raul-Gamarra-jalafund

Reply:
I mean, I don't like seeing a constructor and a field here... but I guess this is the already decided pattern, right?

Reply:
As I mentioned before, it is defined by the team in that way